src/Transformer.php

```diff
@@ -11,14 +11,20 @@
 
     private $key;
     private $dataToTransform;
+    protected $options;
 
-    public function __construct($dataToTransform, $key = null)
+    public static function transform($dataToTransform, $key = null, array $options = [])
```

Comment:
Maybe we should flip key and options, as in this method options will be used more than key. Don't you think so?

Reply:
I have no opinion on the subject :)